db/migrate/20200508121955_create_questions.rb

```diff
@@ -0,0 +1,9 @@
+class CreateQuestions < ActiveRecord::Migration[6.0]
+  def change
+    create_table :questions do |t|
+      t.text :question_text
```

Comment:
не стоит в названиях аттрибутов дублировать имя класса
```suggestion
      t.text :text
```
или
```suggestion
      t.text :body
```